app/controllers/users/webauthn_setup_controller.rb

```diff
@@ -112,6 +121,7 @@ def handle_remember_device
     def process_invalid_webauthn(form)
       if form.name_taken
         flash.now[:error] = t('errors.webauthn_setup.unique_name')
+        # TODO: Retain "platform" query parameter
```

Comment:
This is going to be tricky. This will render the form, but not change the URL. We'd need a redirect to change the URL which I don't think we want. For this to happen we may be to add the platform authenticator param to the URL in the form.

Reply:
Alternatively, if we used a value from the markup instead of the URL, we could set that ivar in this action and it would appear in the markup when rendered.